corehq/apps/case_search/models.py

```diff
@@ -46,6 +45,33 @@ class Meta(object):
         unique_together = ('domain', 'case_type')
 
 
+class RemoveCharacters(models.Model):
```

Comment:
yeah that's tough, maybe `Ignore` is more useful than `Remove`? `IgnorePatterns`?